### PR TITLE
Ledger Wallet CLI Guide (dEBRUYNE)

### PR DESCRIPTION
**Pull request by el00ruobuob**
This is a new user-guide based on @dEBRUYNE-1 [How do I generate a Ledger Monero wallet with the CLI (monero-wallet-cli)?](https://monero.stackexchange.com/questions/8503/how-do-i-generate-a-ledger-monero-wallet-with-the-cli-monero-wallet-cli) StackExchange Q&A, as i told i'll do on [twitter](https://twitter.com/boubour_/status/1004106259937849344).

A few corrections have been applied :
+ De-personalized (s/"I"/"We"/g)
+ Download links moved to getmonero.org instead of GitHub
+ Typo Correction
  - Mac OS referring to GUI 0.12.1.0 instead of CLI 0.12.2.0
  - Monero v0.12.1.0 mentioned instead of v0.12.2.0
  - s/cavaet/caveat/g
+ Precision on installation section of Ledger Monero app guide for windows
+ Update the "need assistance" final note to link to the StackExchange answer.

And i already translated it in French.

### CODE REVIEW

**Comment by el00ruobuob** [comment]
Tested ok on the following platforms:

  | Edge | IE 11 | Chrome | Firefox | Opera | Vivaldi
-- | -- | -- | -- | -- | -- | --
Windows 7 | - | OK | OK | OK | OK | OK
Windows 10 | OK | OK | OK | OK | OK | OK
Ubuntu 18.04 | - | - | OK | OK | OK | OK
CentOS 7 | - | - | - | OK | OK | OK

**Comment by el00ruobuob** [comment]
pinging reviewers @mattcode55 @MaxXor, translation team @erciccione, author @dEBRUYNE-1

**Comment by el00ruobuob** [comment]
Updated to today's edit as mention on [Reddit](https://www.reddit.com/r/Monero/comments/8op6cp/ledger_cli_guides_requires_cli_v01220/e07d3x3/)

**Comment by el00ruobuob** [comment]
And pinging the awful-typos-catchers for French translation @assylias @cryptochangements34 
I did my best ;)

**Comment by el00ruobuob** [comment]
Rebased.

**Comment by MaxXor** [comment]
I don't have Ledger wallet therefore I can not test this.

**Comment by el00ruobuob** [comment]
Me neither @MaxXor i think we'll have to trust @dEBRUYNE-1 on this.
I may buy one later for future updates (if so i will account a part on my next FFS)

**Comment by el00ruobuob** [comment]
Rebased.

File: _i18n/fr/resources/user-guides/ledger-wallet-cli.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+5. Saisissez maintenant :
+
+`--generate-from-device <new-wallet-name> --subaddress-lookahead 3:200`
+
+Remarquez qu'il s'agit simplement d'un emplacement pour le nom effectif du portefeuille. Si vous vouliez, par exemple, nommer votre portefeuille `MoneroWallet`, la commande serait la suivante :
+
+`--generate-from-device MoneroWallet --subaddress-lookahead 3:200`
+
+Note that aforementioned text will be appended to the path of `monero-wallet-cli`. Thus, before you hit enter, your terminal should look like:
```
Comment (cryptochangements34) [inline _i18n/fr/resources/user-guides/ledger-wallet-cli.md]: Translation missing

File: _i18n/fr/resources/user-guides/ledger-wallet-cli.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
+2. Si vous utilisez un nœud distant, ajoutez l'option `--daemon-address host:port` à la commande de l'étape 3 (Windows), l'étape 5 (Mac OS X) ou l'étape 3 (Linux).
+
+3. Si vous les souhaitez, vous pouvez personnaliser la valeur de l'option `--subaddress-lookahead`. La première valeur correspond au nombre de comptes et la seconde au nombre de sous-adresses par compte. ainsi, si par exemple vous voulez pré-générer 5 comptes avec 100 sous-adresses chacun, utilisez `--subaddress-lookahead 5:100`. Gardez à l'esprit que plus vous pré-générez de sous-adresses, plus le temps de création de votre portefeuille sur le Ledger sera long.
+
+4. Vous n'avez à utiliser l'option `--generate-from-device` qu'une seule fois (c-à-d lors de la création du portefeuille). Ultérieurement vous l'utiliserez d'une façon similaire à la CLI habituelle, c'est à dire :
+   1. Vous assurer que votre Ledge est connecté et que l'application Monro est démarrée ;
+   2. Ouvrir `monero-wallet-cli`.
+   3. Entrer le nom du portefeuille de votre portefeuille Monero Ledger ;
```
Comment (cryptochangements34) [inline _i18n/fr/resources/user-guides/ledger-wallet-cli.md]: This sounds repetitive. How about just `Entrer le nom de votre portefeuille Monero Ledger`?

File: _i18n/fr/resources/user-guides/ledger-wallet-cli.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+   4. Entrer le mot de passe pour ouvrir le portefeuille.
+
+   Si les fichiers du portefeuille Ledger ne sont pas dans le même répertoire que `monero-wallet-cli`, vous devrez ouvrir `monero-wallet-cli` avec l'option `--wallet-file /chemin/vers/fichier/clefs.portefeuille`. Sinon, vous pouvez copier les fichiers du portefeuille Ledger dans le même répertoire que `monero-wallet-cli`.
```
Comment (cryptochangements34) [inline _i18n/fr/resources/user-guides/ledger-wallet-cli.md]: vous devez, not devrez

File: _i18n/fr/resources/user-guides/ledger-wallet-cli.md (hunk 1 above)
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/ledger-wallet-cli.md]: ok.

File: _i18n/fr/resources/user-guides/ledger-wallet-cli.md (hunk 2 above)
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/ledger-wallet-cli.md]: ok

File: _i18n/fr/resources/user-guides/ledger-wallet-cli.md (hunk 3 above)
Comment (el00ruobuob) [reply _i18n/fr/resources/user-guides/ledger-wallet-cli.md]: ok

**Comment by el00ruobuob** [comment]
Thanks @cryptochangements34, corrections applied.

**Comment by el00ruobuob** [comment]
Not conflicting, but rebased.

**Comment by el00ruobuob** [comment]
@dEBRUYNE-1 do you approve?

**Comment by dEBRUYNE-1** [comment]
@el00ruobuob - It's OK as long as you ensure it's consistent with the latest version on the Monero SE.

**Comment by luigi1111** [comment]
And rebase. :)

**Comment by el00ruobuob** [comment]
rebased it is sir!

**Comment by el00ruobuob** [comment]
ready to merge @luigi1111

**Comment by el00ruobuob** [comment]
Ok, i checked the consistency. Only change was the `--restore-height` removed. Now i solemnly swear it's up to date and therefor could be merge @luigi1111

**Pull request merged**